homeassistant/components/mqtt/light/schema_template.py

```diff
@@ -252,7 +252,7 @@ def state_received(msg):
                 except ValueError:
                     _LOGGER.warning("Invalid color value received")
 
-            if self._templates[CONF_COLOR_TEMP_TEMPLATE] is not None:
+            if self._templates[CONF_WHITE_VALUE_TEMPLATE] is not None:
```

Comment:
Is there no test for this?

Reply:
Yes there is a test, but the problem is that the test case has both the white_value_template and a color_temp_template set so this copy-paste-error was not found.
Maybe the test could be rewritten to be parametrized, and loop over a list of templates.